tools/c7n_azure/c7n_azure/resources/web_app.py

```diff
@@ -58,6 +60,20 @@ class WebApp(ArmResourceManager):
                 threshold: 1000
                 timeframe: 72
 
+    :example:
+
+    This policy will find all web apps with minimum TLS encryption version not equal to 1.2
+
+    .. code-block:: yaml
+
+        policies:
+          - name: webapp-min-tls-enforcement
+            resource: azure.webapp
+            filters:
+              - type: configuration
+                key: minTlsVersion
+                value: '1.2'
+                op: ne
```

Comment:
I think to do this filter more realistically you might add `value_type: integer` and the op would be `less-than` - aka "find all app services that allow TLS encryption version less than X"

Reply:
I didn't even realize that `value_type` existed. But this leads to a new problem, value_type does not have a `float` or `number` option. So if I do integer here all of the values get rounded to 1 which always satisfies the filter of less than 1.2.

I'll open an issue to add that functionality